packages/frontend/src/components/NavBar/DesktopDrawer.tsx

```diff
@@ -16,27 +16,29 @@ import { Link } from 'react-router-dom';
 import { Routes } from '../../constants/Routes';
 
 export const DesktopDrawer = () => {
-  const { openDrawer, toggleDrawer } = useContext(AppBarContext);
+  const { openDesktopDrawer, toggleDesktopDrawer } = useContext(AppBarContext);
   const { resetMenu } = useContext(MenuContext);
 
-  const toggleDesktopDrawer = () => {
-    if (openDrawer) {
+  const toggleDesktopDrawerr = () => {
```

Comment:
It's generally not recommended to use very similar names for different functions due to confusion, alternative names could be "checkAndToggleDesktopDrawer" or "resetMenuAndToggleDrawer"?

Reply:
Oops, that's actually my mistake when I was just testing it out. Gonna fix that.